docs/README.md

```diff
@@ -83,41 +83,47 @@ The Notebooks can run on various compute targets; the ones currently supported o
 
 # Productionalization
 
-Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. One of the main reasons being that feature engineering (featurization) requires not only new, but also historic data. For reasons outlined below, this solution was built to support real-time featurization and scoring.
+Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. As it is often the case when dealing with time series, feature engineering (featurization) requires access to historical data in addition to the new data.
+
+Generally, predictions can be made either in real time or on a batch basis. This solution was built to support near real-time featurization and scoring, which means up-to-date predictions are generated almost as soon as new data is available.
```

Comment:
If we're saying that predictions can be made in real-time or batch, why are we then saying near-real-time?  Just stick with real-time.

Predictions can be made either in real-time or in batches. This solution was built to support real-time featurization and scoring, which means predictions are generated when new data is available.